### PR TITLE
fix: Add Cargo.lock if not ignored

### PR DESCRIPTION
**Pull request by badboy**
Step 1 for #59

We still need to make sure `cargo.lock` reflects the latest changes.

### CODE REVIEW

**Comment by badboy** [comment]
Tests broken because I don't know

**Comment by badboy** [comment]
This looks bigger than it is. The actual changes are only in files in `src/`, so basically the first 4 changes listed in the diff view.

**Comment by badboy** [comment]
@schultyy Wanna take a look?

**Comment by schultyy** [comment]
Yep, I have a look :)

File: src/cargo.rs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+pub fn package(repository_path: &str) -> bool {
+    let manifest_path = format!("{}/Cargo.toml", repository_path);
+    Command::new("cargo")
+        .arg("package")
+        .arg("--manifest-path")
+        .arg(manifest_path)
+        .status()
+        .map(|s| s.success())
+        .unwrap_or(false)
```
Comment (schultyy) [inline src/cargo.rs]: Don't we want to know why it failed in case of failure?

Comment (badboy) [inline src/cargo.rs]: That's why we're returning false. Not really proper error handling (yet), but we still abort in main if we get `false` for now

**Pull request merged**